packages/TelegramClient-UI.package/TCUEmojiHelper.class/class/createUnicodeRunsFrom..st

```diff
@@ -0,0 +1,8 @@
+unicode matching
+createUnicodeRunsFrom: aString
```

Comment:
We already mentioned the `run...` problem we see in many places. What do you think about calling this method `splitCharsAndEmojiCodesFrom:`. (The variables in the block coulld the be called `emojiCodesAndChars` and `char`.